rollup/sync_service.go

```diff
@@ -640,5 +642,92 @@ func (s *SyncService) ApplyTransaction(tx *types.Transaction) error {
 	if err != nil {
 		return fmt.Errorf("invalid transaction: %w", err)
 	}
+
+	// Set the raw transaction data in the meta
+	txRaw, err := getRawTransaction(*tx)
```

Comment:
Assuming the upgrade to use standard RLP serialization, this will be able to be moved

Reply:
I just tried this branch against my WIP encoding PR (https://github.com/ethereum-optimism/contracts/pull/300/files) and replaced `getRawTransaction(*tx)` with `rlp.EncodeToBytes(tx)` and it worked first try. Woot!